ui/app/models/ldap/role.js

```diff
@@ -224,9 +230,11 @@ export default class LdapRoleModel extends Model {
   }
 
   fetchCredentials() {
-    return this.store.adapterFor('ldap/role').fetchCredentials(this.backend, this.type, this.name);
+    return this.store
+      .adapterFor('ldap/role')
+      .fetchCredentials(this.backend, this.type, this.completeRoleName);
```

Comment:
This is the bug fix when clicking the popup menu quick actions for an ldap role, such as "Get credentials"
<img width="1156" alt="Screenshot 2025-01-06 at 11 19 28 AM" src="https://github.com/user-attachments/assets/062ec00c-1835-4d94-b73d-2d52eb582a5d" />